pkg/analysis/utils/utils.go

```diff
@@ -77,6 +77,21 @@ func LookupTypeSpec(pass *analysis.Pass, ident *ast.Ident) (*ast.TypeSpec, bool)
 	return nil, false
 }
 
+// FieldName returns the name of the field. If the field has a name, it returns that name.
+// If the field is embedded and it can be converted to an identifier, it returns the name of the identifier.
+func FieldName(field *ast.Field) string {
```

Comment:
Include in the GoDoc that if the field name cannot be determined that it returns the empty string (`""`)?

Reply:
Thanks, I'll add it.